docs/classad-attributes/job-classad-attributes.rst

```diff
@@ -301,111 +204,71 @@ all attributes.
 
         CommittedTime - CommittedSuspensionTime
 
+:index:`SYSTEM_JOB_MACHINE_ATTRS`
 
-:index:`CommittedSlotTime<single: CommittedSlotTime; ClassAd job attribute>`
-:index:`job ClassAd attribute<single: job ClassAd attribute; CommittedSlotTime>`
-
-
-``CommittedSlotTime``
+:classad-attribute:`CommittedSlotTime`
     This attribute is identical to ``CommittedTime`` except that the
     time is multiplied by the ``SlotWeight`` of the machine(s) that ran
     the job. This relies on ``SlotWeight`` being listed in
     ``SYSTEM_JOB_MACHINE_ATTRS``
```

Comment:
I'm OK with removing the index entry for `SYSTEM_MACHINE_JOB_ATTRS`, but think about why you didn't for the other place a config knob shows up and make sure we're being consistent.  Also, as long as we're here, maybe make this :macro:`SYSTEM_JOB_MACHINE_ATTRS`?

Reply:
Never mind.  I missed that you just moved it up to before the entry.